docs/source/using/troubleshooting.adoc

```diff
@@ -213,3 +213,11 @@ You can also use the GUI to add yourself to the Hyper-V Administrators group as
 Now you can run the Hyper-V commands as a normal user.
 
 For more options for Hyper-V see https://blogs.msdn.microsoft.com/virtual_pc_guy/2010/09/28/creating-a-hyper-v-administrators-local-group-through-powershell[creating Hyper-V administrators local group].
+
+== Hyper-V: Minishift running with Hyper-V fails when connected to OpenVPN
+
+If you try to use Minishift with Hyper-V using an external virtual switch, while you are connected to a VPN such as OpenVPN, Minishift might fail to provision the VM.
+
+Cause: Hyper-V networking may not route the network traffic in both directions properly, when connected to a VPN.
```

Comment:
@robin-owen what happened to this entry?

Reply:
Found it, https://docs.okd.io/latest/minishift/troubleshooting/troubleshooting-driver-plugins.html#hyperv-fails-openvpn But this actually applies to all platforms